lib/system/web/src/closure.rs

```diff
@@ -0,0 +1,60 @@
+use crate::prelude::*;
```

Comment:
this module does not implement `closure` - it implements some utils for it. In fact it implements some kind of storage. I would move it to `src/closure/storage` instead. Right now, after browisng the filenames I was sure thisis an altenrative to wasm-bindgen implementation for a closure

Reply:
done